elastic/translate.go

```diff
@@ -58,44 +66,48 @@ func encodeParam(concept string, param map[string]interface{}) (map[string]inter
 	case "lte":
 		l = &LessThanTerm{conceptPath, v, true}
 	case "range":
-		s := v.([]interface{})
-		gt := s[0]
-		lt := s[1]
-		l = &RangeTerm{conceptPath, lt, gt}
+		s, ok := v.([]interface{})
+		if !ok {
+			return nil, fmt.Errorf("Cannot parse 'range' query value to interface slice: %v", v)
+		}
+		l = &RangeTerm{conceptPath, s[1], s[0]}
```

Comment:
I overlooked this before, but this depends on the length of the slice to be two..